cmd/crc/cmd/daemon_windows.go

```diff
@@ -0,0 +1,35 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/code-ready/crc/pkg/crc/api"
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(daemonCmd)
+}
+
+var daemonCmd = &cobra.Command{
```

Comment:
this code is duplicated between Windows and unix. When using a nonwindows.go file we can use the daemon.go for the shared code.

Reply:
Done, thanks!